web/cache.go

```diff
@@ -38,20 +38,26 @@ type cacheEntry []byte
 
 func (c *cache) newCacheEntry() cacheEntry {
 	ce := make([]byte, widthPermalink+encoding.WidthTime+1)
-	permalink := uuid.NewRandom()
-	copy(ce[idxPermalink:], permalink)
+	permalink := uuid.New()
+	copy(ce[idxPermalink:], permalink[:])
```

Comment:
A little weird to convert from an array to a slice to an array here...but I think the alternative is to make `cacheEntry` an array - `[16]byte`